docs/topics/request-response.rst

```diff
@@ -841,6 +841,16 @@ HtmlResponse objects
 
 .. _meta http-equiv: https://www.w3schools.com/TAGS/att_meta_http_equiv.asp
 
+JsonResponse objects
+--------------------
+
+.. class:: JsonResponse(url[, ...])
+
+    The :class:`JsonResponse` class is a subclass of :class:`TextResponse` which
+    adds a :func:`json` function that decodes the JSON body and returns
+    appropriate Python objects.
+
```

Comment:
You are still missing the method documentation here. Add `.. automethod:: json` indented 1 level deeper than `.. class`.